src/lib/api-v1.ts

```diff
@@ -113,6 +113,24 @@ export default class ApiV1 {
     return this.put(`/workflows/${args.workflowKey}/activate`, {}, { params });
   }
 
+  async runWorkflow({
+    args,
+    flags,
+  }: Props): Promise<AxiosResponse<ActivateWorkflowResp>> {
+    const params = prune({
+      environment: flags.environment,
+      recipients: flags.recipients,
+      tenant: flags.tenant,
+      // For now, we assume the data passed by the user is stringified
+      // because it will be passed through the flags. When we allow users
+      // to point to a file for the data we can rework this.
```

Comment:
I think this comment is no longer applicable because `jsonStr` flag automatically parses the data string?